src/data/migrations/20190822155403_create_steps_table.js

```diff
@@ -0,0 +1,25 @@
+exports.up = function(knex) {
+    return knex.schema.createTable("steps", steps => {
+        steps.increments()
+        steps
+            .integer("number")
+            .unsigned()
+            .notNullable()
+        steps.text("description").notNullable()
```

Comment:
The description here for steps wasn't in our original schema but I think it could be useful, what do you think?